addons/cookoff/functions/fnc_handleDamage.sqf

```diff
@@ -16,13 +16,13 @@
 #include "script_component.hpp"
 
 params ["_simulationType", "_thisHandleDamage"];
-_thisHandleDamage params ["_vehicle", "", "_damage", "_source", "_ammo", "_hitIndex", "_shooter"];
+_thisHandleDamage params ["_vehicle", "", "_damage", "_source", "_ammo", "_hitIndex", "_shooter", "_hitPoint"];
 
 // it's already dead, who cares?
 if (damage _vehicle >= 1) exitWith {};
 
-// get hitpoint name
-private _hitpoint = "#structural";
```

Comment:
`"#structural"` is a special var that would be lost by this
you might be able to replace line 28, but IIRC there might be some problem with sub-turret hitpoints